tests/test_handwritten.py

```diff
@@ -0,0 +1,25 @@
+import unittest, os
+from nelox.parser import Parser
+from nelox.scanner import Scanner
+
+class TestHandWritten(unittest.TestCase):
+    base = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+        "dataset", "handwritten"
+    )
+
+    def running(self, filename):
+        path = os.path.join(self.base, filename)
+        with open(path, "r") as f:
+            src = f.read()
+        parser = Parser(Scanner(src))
+        parser.parse()
+
+    def test_all_cases(self):
+        for i in range(1, 13):
```

Comment:
Can you have it loop over the files instead of specifying a number?  It's going to be annoying to update otherwise.